code-style/php/README.md

```diff
@@ -0,0 +1,169 @@
+# PHP Coding Standards
+![](php-logo.png)
+
+Despite still being one of the most used languages to build web applications, PHP doesn't have the best reputation. There are a number of frameworks we use that are built with PHP. In order to avoid PHP's reputation of messy codebases, we should utilize [PHP_CodeSniffer](phpcs) to detect violations of a defined coding standards.
+
+### What Standard Should I use?
+
+Many frameworks have their own standards declared. We should use the framework's defined standard when possible. For instance, when developing a WordPress theme, you should utilize the [WordPress Standard](wp). If there is not a standard defined, or you are not using a framework, you should use the [PSR-2 Standard](psr-2).  Some standards are:
+
+- [PSR-2 / Default](psr-2)
+- [WordPress](wp)
+- [Symfony](symfony)
+- [Drupal](drupal)
+
+### Enforce my standard with PHPCS
+
+[PHP_CodeSniffer](phpcs) allows us to lint our PHP. To install PHPCS:
+
+1. Install PHP CodeSniffer:
+
+    `composer global require "squizlabs/php_codesniffer=*"`
+1. You need access to global composer installs. In your .zshrc file, ensure you have:
+
+    `export PATH=~/.composer/vendor/bin:$PATH`
+
+#### Install Coding Standards:
+
+<details>
+  <summary>PSR-2 Coding Standard</summary>
+  <p>PSR-2 is installed by default with phpcs</p>
+</details>
+
+<details>
+  <summary>Drupal Coding Standard</summary>
+  <ol>
+    <li>Install Drupal Standard
+      <pre>composer global update drupal/coder --prefer-source</pre>
+    </li>
+    <li>Register Drupal standards in PHP Codesniffer
+      <pre>composer global require drupal/coder:^8.3.1</pre>
+      <pre>composer global require dealerdirect/phpcodesniffer-composer-installer</pre>
+    </li>
+  </ol>
+</details>
+
+<details>
+  <summary>Symfony Coding Standard</summary>
+  <p>Symfony follows PSR-2 with a few additional standards.</p>
+  <ol>
+    <li>Install the coding standard as a dependency of your project
+      <pre>composer require --dev escapestudios/symfony2-coding-standard:3.x-dev</pre>
+    </li>
+    <li>Add the coding standard to the PHP_CodeSniffer install path
+      <pre>phpcs --config-set installed_paths vendor/escapestudios/symfony2-coding-standard</pre>
+    </li>
+  </ol>
+</details>
+
+<details>
+  <summary>WordPress Coding Standard</summary>
+  <ol>
+    <li>Install WordPress Standard
+      <pre>git clone https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards.git ~/wpcs</pre>
+    </li>
+    <li>Register WordPress standards in PHP Codesniffer
+      <pre>phpcs --config-set installed_paths /Users/<your-user>/wpcs</pre>
+    </li>
+  </ol>
+</details>
+
+After adding your standard(s), you should verify them with: `phpcs -i`
```

Comment:
Is `phpcs -i` the command that validates the codebase against the rules? I almost glossed over this part, maybe break it out under a heading like "Run PHPCS in the terminal"? That may make it more clear how to use it for folks not using VSCode or CircleCI integration.

Also, is this intended to be run in the root directory of a project? If so, you might want to mention that as well.